src/main/java/blitz/gameoflife/GameComponent.java

```diff
@@ -0,0 +1,37 @@
+package blitz.gameoflife;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class GameComponent extends JComponent {
+
+    private Grid grid;
+    private int cellSize;
+
+    public GameComponent(Grid grid, int cellSize) {
+        this.grid = grid;
+        this.cellSize = cellSize;
+    }
+
+    @Override
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+
+        g.setColor(Color.WHITE);
+        g.fillRect(0, 0, getWidth(), getHeight());
+
+        int[][] field = grid.getField();
+
+        for (int y = 0; y < field.length; y++) {
+            for (int x = 0; x < field[y].length; x++) {
+                if (field[y][x] == 1) {
+                    g.setColor(Color.BLACK);
+                    g.fillRect(x * cellSize, y * cellSize, cellSize, cellSize);
+                }
+
+                g.setColor(Color.GRAY);
+                g.drawRect(x * cellSize, y * cellSize, cellSize, cellSize);
```

Comment:
Instead of drawing individual rects for each cell, can you figure out how to do it just by drawing grid lines?